Fix:
tclbasic.c:23:10: fatal error: 'tclxtend.h' file not found
#include "tclxtend.h"
         ^~~~~~~~~~~~
--- src/tclx/ucbsrc/makefile.orig	2018-06-29 04:19:31.000000000 -0500
+++ src/tclx/ucbsrc/makefile	2022-05-16 21:01:01.000000000 -0500
@@ -24,7 +24,7 @@ SHELL=/bin/sh

 #------------------------------------------------------------------------------

-CFLAGS=$(OPTIMIZE_FLAG) $(XCFLAGS) -I../$(TCL_UCB_DIR) $(MEM_DEBUG_FLAGS) \
+CFLAGS=$(OPTIMIZE_FLAG) $(XCFLAGS) -I../src -I../$(TCL_UCB_DIR) $(MEM_DEBUG_FLAGS) \
   $(SYS_DEP_FLAGS)

 #------------------------------------------------------------------------------
